### DISCUSSION TITLE
virtual indents for sections?

### DISCUSSION THREAD

**Discussion by hans-d** [magit/magit#4566] [Ideas]
I might perhaps overlooking setting already there...

Currently all the sections created in a eg diff screen are all left aligned
```
Unstaged changes
modified some-file
@@ .....
....
@@ ....
...
```

I'm looking for a way to display it a bit more indented/tree like, eg
```
Unstaged stanges
  modified some-file
    @@ .....
      ....
    @@ ....
      ...
```
Where also the `magit-diff-....-highlight` takes into account the adjusted virtual left margin

**Comment by tarsius** [comment]
Such a feature does not exist and I do not intend to implement it. Sorry.

**Reply by hans-d** [reply]
fair enough.
